frontend/src/components/ErrorBoundary/ErrorFallback.tsx

```
@@ -0,0 +1,176 @@
+            </p>
+          </div>
+        </div>
+
+        {/* Error Details (Dev Mode or when showDetails is true) */}
+        {(isDev || showDetails) && (
+          <div className="mt-4">
+            <button
+              onClick={() => setIsExpanded(!isExpanded)}
+              className="text-sm text-blue-600 hover:text-blue-800 font-medium focus:outline-none focus:underline"
+            >
+              {isExpanded ? 'Hide' : 'Show'} Error Details
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)